docs/source/releases/v0_1_0.rst

```diff
@@ -0,0 +1,81 @@
+    # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+    # # #
+    # # # Author:
+    # # # Naval Research Laboratory, Marine Meteorology Division
+    # # #
+    # # # This program is free software: you can redistribute it and/or modify it under
+    # # # the terms of the NRLMMD License included with this program. This program is
+    # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    # # # for more details. If you did not receive the license, for more information see:
+    # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Version 0.1.0 (YYYY-MM-DD)
```

Comment:
@cpcamacho @jsolbrig I decided to move CHANGELOG.rst to valid release notes that will get built properly when we get the documentation builds to work.  I could be talked out of this... (And just move this file back up to CHANGELOG.rst)

Reply:
This is fine for now. I think. I'd really like to figure out a way to make the release notes easier for people to put together. I still don't know how, though. The release notes are by far the most confusing part of making a PR, even for me, and especially for a new contributor.

Reply:
I put the reference CHANGELOG.rst back, since that is typically what the repos look like (referencing CHANGELOG_TEMPLATE.rst and docs/source/releases).

Then we can modify CHANGELOG_TEMPLATE.rst with updated instructions in the future, and the plugin repositories will still point to the same place.  I'm going to resolve.